app/assets/stylesheets/admin/views/_translations.scss

```
@@ -0,0 +1,4 @@
+.app-view-edition-translation__english-content,
+.app-view-person-translation__english-content {
+  color: $govuk-secondary-text-colour;
+}
```

I think you've done the right thing here by consolidating these rules into one place.

You might want to go a step further and just define them as one class.

```suggestion
.app-view-translation__english-content {
  color: $govuk-secondary-text-colour;
}
```

..and update the HTML views accordingly.

---

yea sounds good. Got a view on how is should be structured to be BEM compliant? I'm never really sure where something like this should live? It's not really an override & it is shared across many views

---

cool yea that sounds good to me. It's still super intuitive what it does and where to look for it that way. I'll action that before merge.

Thanks for the speedy response!